frontend/components/Domain/SearchFilter.vue

```diff
@@ -11,28 +11,42 @@
       <v-card width="400">
         <v-card-text>
           <v-text-field v-model="state.search" class="mb-2" hide-details dense :label="$tc('search.search')" clearable />
-          <v-switch
-            v-if="requireAll != undefined"
-            v-model="requireAllValue"
-            dense
-            small
-            :label="`${requireAll ? $tc('search.has-all') : $tc('search.has-any')}`"
-          >
-          </v-switch>
+          <div class="d-flex py-4">
+            <v-btn
+              small
+              color="accent"
+              class="mr-2 my-auto"
+              @click="clearSelection"
+            >
+              {{ $tc("search.clear-selection") }}
+            </v-btn>
+            <v-switch
+              v-if="requireAll != undefined"
+              v-model="requireAllValue"
+              dense
+              small
+              hide-details
+              class="ml-2 my-auto"
+              :label="`${requireAll ? $tc('search.has-all') : $tc('search.has-any')}`"
+            />
+          </div>
```

Comment:
Love the clear selection button 😍 
i think having it on the right side would separate it better from the any/all button

```suggestion
          <div class="d-flex py-4">
            <v-switch
              v-if="requireAll != undefined"
              v-model="requireAllValue"
              dense
              small
              hide-details
              class="my-auto"
              :label="`${requireAll ? $tc('search.has-all') : $tc('search.has-any')}`"
            />
            <v-spacer />
            <v-btn
              small
              color="accent"
              class="mr-2 my-auto"
              @click="clearSelection"
            >
              {{ $tc("search.clear-selection") }}
            </v-btn>
          </div>
```